influxdb3-setup.sh

```
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+if [ "$INFLUXDB3_START" = "true" ]
+then
+    docker run -it -p 8181:8181 influxdb3-core serve --object-store memory --writer-id writer0
+    influxdb3 create $INFLUXDB3_DATABASE
+
+    if [ "$INFLUXDB3_CREATE_TOKEN" = "true" ]
+    then
+        INFLUXDB3_TOKEN=$(influxdb3 create token | grep "Token:" | awk '{print $2}' | head -n 1)
```

Shellcheck (0.10.0)

[warning] INFLUXDB3_TOKEN appears unused. Verify use (or export if used externally).

(SC2034)